beesdoo_pos/models/beesdoo_pos.py

```diff
@@ -4,13 +4,11 @@
 class BeescoopPosPartner(models.Model):
     _inherit = "res.partner"
 
-    def _get_eater(self):
-        eaters = [False, False, False]
-        for i, eater in enumerate(self.child_eater_ids):
-            eaters[i] = eater.name
-        return tuple(eaters)
-
     @api.multi
     def get_eater(self):
-        eater1, eater2, eater3 = self._get_eater()
-        return eater1, eater2, eater3
+        self.ensure_one()
+        eaters = []
+        # todo check for max eater
+        for eater in self.child_eater_ids:
+            eaters.append(eater.name)
+        return eaters
```

Comment:
a shorter way of writing this:

```suggestion
        return [eater.name for eater in self.child_eater_ids]
```

Reply:
Nice thanks :)